src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const countMale = people.filter(el => el.sex === 'm');
+
+  const result = century ? (
+    countMale
+      .filter(cent => Math.ceil(cent.died / 100) === century)
+  ) : countMale;
```

Comment:
const result = century 
      ? countMale .filter(cent => Math.ceil(cent.died / 100) === century
      : countMale;

Reply:
I think you should format all of the ternary operators in your code